presto-elasticsearch/src/test/java/io/prestosql/elasticsearch/TestElasticsearchIntegrationSmokeTest.java

```diff
@@ -23,11 +23,10 @@
 import org.apache.http.HttpHost;
 import org.apache.http.entity.ContentType;
 import org.apache.http.nio.entity.NStringEntity;
-import org.elasticsearch.action.index.IndexRequest;
-import org.elasticsearch.action.support.WriteRequest;
 import org.elasticsearch.client.RestClient;
 import org.elasticsearch.client.RestHighLevelClient;
 import org.intellij.lang.annotations.Language;
+import org.testcontainers.shaded.com.fasterxml.jackson.databind.ObjectMapper;
```

Comment:
Don't import the ObjectMapper from the shaded version of jackson inside testcontainers. Add a dependency for jackson to the module, instead.

Reply:
I dont know why is importing that, jackson is a dependency of this module

fixed